num_wheels: 4
device: enp2s0

#vlin_max: 1.2  # m / s
#va_max: 1.1  # rad / s
#vlin_acc_max: 0.4  # m / s^2
#vlin_dec_max: 1.0  # m / s^2
#va_acc_max: 0.5  # rad / s^2
#va_dec_max: 1.0  # rad / s^2

modules:
  platform_driver:
    type: platform_driver
    controller: velocity_platform_controller
  power_management_unit:
    type: power_management_unit
    ethercat_number: 1

wheel0:
  ethercat_number: 3
  x: 0.188
  y: 0.2075
  a: 0

wheel1:
  ethercat_number: 4
  x: -0.188
  y: 0.2075
  a: 0

wheel2:
  ethercat_number: 6
  x: -0.188
  y: -0.2075
  a: 0

wheel3:
  ethercat_number: 7
  x: 0.188
  y: -0.2075
  a: 0
